lib/components/Forbidden403/Forbidden403.tsx

```diff
@@ -0,0 +1,21 @@
+import { Section } from '../../layouts/Section/Section'
+import { PageHeader } from '../PageHeader/PageHeader'
+
+export interface Forbidden403Props {
+  children?: React.ReactNode
+  content?: string
```

Comment:
The content should be standardized, not a prop. Remove the content prop, we'll hardcore the content

Reply:
done